cleanlab/object_detection/filter.py

```diff
@@ -42,8 +42,9 @@
 def find_label_issues(
     labels: List[Dict[str, Any]],
     predictions: List[np.ndarray],
-    *,
     return_indices_ranked_by_score: bool = False,
```

Comment:
`return_indices_ranked_by_score` param should be optional as well.

Reply:
@aditya1503 why do you keep moving the * arguments? please never move them up above optional args